src/rust/engine/fs/src/lib.rs

```diff
@@ -463,25 +463,26 @@ pub struct PathGlobs {
 
 impl PathGlobs {
   pub fn create(
-    include: &[String],
-    exclude: &[String],
+    globs: &[String],
```

Comment:
This PR's overall impact seems actually much higher for rust usability than python, since rust doesn't have default arguments like python methods can!

Reply:
Good point!